### PR TITLE
Document the DML operations support on GCS

### PR DESCRIPTION
**Pull request by findinpath**
## Description

Left-over docs change for Delta DML operation support on GCS from https://github.com/trinodb/trino/pull/14766

## Non-technical explanation

N/A

## Release notes

(x) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: docs/src/main/sphinx/connector/delta-lake.rst [hunk 1]
```diff
@@ -566,6 +566,13 @@ Write operations are supported for tables stored on the following systems:
   detects write collisions on ADLS Gen2 when writing from multiple Trino
   clusters, or from as any number of open source Delta Lake clusters.
 
+* Google Cloud Storage
```
Comment (colebow) [inline docs/src/main/sphinx/connector/delta-lake.rst:569]: If this is treated the same as Azure ADLS, we can probably combine them into one bullet point, e.g. `* Azure ADLS Gen2, Google Cloud Storage`

Then "Writes to the Azure ALDS Gen2 and Google Cloud Storage file systems are enabled..."

File: docs/src/main/sphinx/connector/delta-lake.rst [hunk 2]
```diff
@@ -566,6 +566,13 @@ Write operations are supported for tables stored on the following systems:
   detects write collisions on ADLS Gen2 when writing from multiple Trino
   clusters, or from as any number of open source Delta Lake clusters.
```
Comment (colebow) [inline docs/src/main/sphinx/connector/delta-lake.rst:567]:
```suggestion
  detects write collisions on ADLS Gen2 when writing from multiple Trino
  or Delta Lake clusters.
```
I can't figure out what this is trying to say and I don't think we need to tell people that Delta Lake is open source here, so let's clean it up.

Comment (findinpath) [reply docs/src/main/sphinx/connector/delta-lake.rst]: I rewrote a bit this statement to be a bit more concise, but still tried to keep its essence.

**Review by colebow — APPROVED** [review]

**Review by ebyhr — APPROVED** [review]

File: docs/src/main/sphinx/connector/delta-lake.rst [hunk 3]
```diff
@@ -560,11 +560,11 @@ You can use the connector to :doc:`/sql/insert`, :doc:`/sql/delete`,
 
 Write operations are supported for tables stored on the following systems:
 
-* Azure ADLS Gen2
+* Azure ADLS Gen2, Google Cloud Storage
 
-  Writes to the Azure ADLS Gen2 file system are enabled by default. Trino
-  detects write collisions on ADLS Gen2 when writing from multiple Trino
-  clusters, or from as any number of open source Delta Lake clusters.
+  Writes to the Azure ADLS Gen2 and Google Cloud Storage file systems are
```
[the remark this answers is not in the document]
Comment (findinpath) [reply docs/src/main/sphinx/connector/delta-lake.rst:565]: `file systems` -> `storage systems` ?

[the remark this answers is not in the document]
Comment (ebyhr) [reply docs/src/main/sphinx/connector/delta-lake.rst:565]: Or just `storage` looks acceptable to me.

**Pull request merged**